include/quicr/hex_endec.h

```diff
@@ -65,7 +65,8 @@ class HexEndec
     template<Unsigned... UInt_ts>
     static inline std::string Encode(std::span<uint16_t> distribution, UInt_ts... values)
     {
-        static_assert(Size >= (Dist + ...), "Total bits cannot exceed specified size");
+        if(Size < std::accumulate(distribution.begin(), distribution.end(), 0))
```

Comment:
So dist of bits doesn't have to total size?

Reply:
No, you're allowed to let the rest of it be filled with zeros. The only constraint in this Encode is that you can't exceed Size. The remaining bits will be 0s (numero-uri uses this functionality).